### ISSUE TITLE
QEs with embedded Compendium Entries don't run

### ISSUE THREAD

**Issue by spetzel2020** [spetzel2020/quick-encounters#116]
Foundry v10 & QE 1.1.0

You can drag the Compendium entry to the underlying JE and it appears in the QE. However, when you press the [Run QE] button you get:

```
Uncaught (in promise) TypeError: Cannot convert undefined or null to object
    at Actors.fromCompendium (foundry.js:14142:18)
    at Actors.fromCompendium (foundry.js:14248:24)
    at Actors.importFromCompendium (foundry.js:14110:29)
    at async QuickEncounter.getActor (QuickEncounter.js:1224:25)
    at async QuickEncounter.generateFullExtractedActorTokenData (QuickEncounter.js:1258:27)
    at async QuickEncounter.run (QuickEncounter.js:1098:9)
```

**Comment by spetzel2020** [comment]
The Compendium entry is not shown correctly:
![image](https://user-images.githubusercontent.com/10853466/197068724-34b5ac09-2cc4-407f-bba5-d99f37636e1b.png)

**Comment by spetzel2020** [comment]
Problem occurs in `QESheet.js#computeCombatantsForDisplay()#186` here:
```
                    const entry = index.find(e => e._id === combatant.actorId);
```

**Comment by spetzel2020** [comment]
Fixed in 1.1.0e by stripping off extraneous Compendium name.
Also needs to be changed here:
```
Uncaught (in promise) TypeError: Cannot convert undefined or null to object
    at Actors.fromCompendium (foundry.js:14142:18)
    at Actors.fromCompendium (foundry.js:14248:24)
    at Actors.importFromCompendium (foundry.js:14110:29)
    at async QuickEncounter.getActor (QuickEncounter.js:1224:25)
    at async QuickEncounter.generateFullExtractedActorTokenData (QuickEncounter.js:1258:27)
    at async QuickEncounter.run (QuickEncounter.js:1098:9)
```

**Comment by spetzel2020** [comment]
Fixed in 1.1.0e

**Comment by spetzel2020** [comment]
Released in 1.1.0
